samples/README.md

```diff
@@ -0,0 +1,88 @@
+# JAVA-SDK sample templates
```

Comment:
Make changes to match comments in the parallel Python SDK PR: https://github.com/skyflowapi/skyflow-python/pull/64